dataset-conversion/README.md

```diff
@@ -17,6 +17,18 @@ The data used for training the model are hosted on a private repository accordin
   - Single session
   - Number of subjects: 80
   - Post-operative subjects only
+- University of Paris (`sci-paris`) 🇫🇷
+  - Contrasts used for training: Sagittal T2w
+  - Manual segmentations for both SC and lesion only available for Sagittal T2w
+  - Single session
+  - Number of subjects: 14
+  - Lesion etiology: traumatic lesions
+  - Surgery: Mix of operated and non-operated subjects
+- PRAXIS Spinal Cord Institue 🇨🇦
+  - Sites: 
+  - Number of subjects (total): 84
+  - Lesion etiology: Acute pre-operative traumatic SCI
+  
```

Comment:
@valosekj could you please update this list with info from the PRAXIS dataset? thank you!

Reply:
Done in https://github.com/ivadomed/model_seg_sci/pull/87/files/d8d3b3c3bc704314b5fd925f31f4bbe1ce1c296e#r1670981634. You can accept the commit suggestion if you agree.

Reply:
We should maybe also add `dcm-zurich-lesions-20231115`

Reply:
Done now! https://github.com/ivadomed/model_seg_sci/pull/87/commits/abfdd317c3df122dd543802ac7fd32f8860a5c1c